lib/FrameManager.js

```diff
@@ -761,11 +761,12 @@ class WaitTask {
   /**
    * @param {!Frame} frame
    * @param {Function|string} predicateBody
+   * @param {string} [title] what is being waited for (used in timeout error message)
```

Comment:
nit: drop the comment and square braces around `title`

```js
 * @param {string} title
```

Reply:
Optional parameters are marked with `=` sign, e.g.:

```js
 * @param {string=} title
```

However, I don't think it should be optional.

Reply:
https://esdoc.org/manual/tags.html#optional-and-default

> ```js
> /**
>  * @param {number} [param] - this is optional param.
>  */
> function myFunc(param){...}
>
> /**
>  * @param {number} [param=10] - this is default param.
>  */
> function myFunc(param){...}
```

Reply:
We use closure annotations, there's a link to the manual in [CONTRIBUTING.md](https://github.com/GoogleChrome/puppeteer/blob/master/CONTRIBUTING.md#code-style)

Check out optionals in closure annotations: https://github.com/google/closure-compiler/wiki/Annotating-JavaScript-for-the-Closure-Compiler#optional-parameter-in-a-param-annotation